docs/Reference/Routes.md

```diff
@@ -227,8 +227,23 @@ checked before parametric and wildcard.*
 
 ```js
 // parametric
-fastify.get('/example/:userId', (request, reply) => {})
-fastify.get('/example/:userId/:secretToken', (request, reply) => {})
+fastify.get('/example/:userId', (request, reply) => {
+    /**
+     * curl ${app-url}/example/12345
+     * userId === '12345'
+     */
```

Comment:
Please keep jsdoc delimited comments reserved for actual jsdoc. Replace all `/** */` blocks with simple `//` blocks:

```suggestion
     // curl ${app-url}/example/12345
     // userId === '12345'
```

Reply:
done this